files-community/Files
# RichCommand: QuickLook

**Resolved / Related Issues**
   Closes #issue...

**Validation**
How did you test these changes?
   1. Install and open QuickLook
   2. Open Files and select an item
   3. Press the space bar and see if the QuickLook preview opens
   4. Select a different item with the arrow keys or mouse and see if the preview updates
   5. Select multiple items and press the space bar, verify that the items remain selected.

**Screenshots (optional)**
Add screenshots here.
